examples/routing_service/routing_service_msql_adapter/cpp/RsLog.hpp

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
We could use logging functions from our logger. @titosanxez what do you think?

Reply:
That's a requested feature that will include for next release. Nevertheless, I pointed out in a previous comment that the error handling in this code is not safe. There should not logging but exception throwing instead

Reply:
I use logging for tracing as well as logging errors. I would like to keep it for the tracing, but will look at your comments about exception throwing.